src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,48 @@
+package nextstep.subway.line.domain;
+
+import java.util.Objects;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+
+    private int distance;
+
+    protected Distance() {
+
+    }
+
+    public Distance(int distance) {
+        if (distance < 1) {
+            throw new IllegalArgumentException("거리는 양수값 이어야 합니다.");
+        }
+
+        this.distance = distance;
+    }
+
+    public Distance subtract(Distance newDistance) {
+        return new Distance(this.distance -= newDistance.distance);
```

Comment:
```suggestion
        return new Distance(this.distance - newDistance.distance);
```

기존 distance 값을 수정하는 것은 의도된 내용일까요?